modules/content/github-content.service.ts

```diff
@@ -27,16 +36,14 @@ export class GithubContentService implements ContentService {
     async syncTokenContentData(): Promise<void> {
```

Comment:
would it make sense to sync all the tokens at once without splitting per chain? i'm not sure if we aren't overcomplicating it here, since the whole list is fetched anyways.

Reply:
bit lower there is upsert call, it can take chain as `token.chainId` instead of networkContext